courses/01_beginners/Makefile

```
@@ -1,11 +1,16 @@
-all: main
+handout:
+	latexmk -pdfxe -shell-escape -jobname=handout $<
 
-%:
-	latexmk -pdfxe $<
+presentation:
+	latexmk -pdfxe -shell-escape -usepretex="\def\createHandout{}" -jobname=presentation $<
+
+all: handout presentation
 
 clean:
-	latexmk -c
+	latexmk -c -jobname=handout
+	latexmk -c -jobname=presentation
```

```suggestion
	latexmk -C
```

I would clean everything up, including the compilation files generated from the LaTeX editor (`main.*`).

---

latexmk -c without a jobname doesn't clear files `presentation.*` or `handout.*`.
Current Makefile should not generate any `main.*` files, it must have been rest of a compilation you did before pulling this.

---

Fair.